src/ai/backend/manager/idle.py

```diff
@@ -302,22 +325,67 @@ async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
 
 class UtilizationIdleChecker(BaseIdleChecker):
     """
-    Checks the idleness of a session by the current utilization of all
-    compute devices and agents assigned to it.
+    Checks the idleness of a session by the average utilization of compute devices.
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
+    check_interval: float = 20.0
+
+    _config_iv = t.Dict(
+        {
+            t.Key("time-window", default="10m"): tx.TimeDuration(),
+            t.Key("initial-grace-period", default="5m"): tx.TimeDuration(),
+            t.Key("thresholds-check-operator", default="and"): t.String,
```

Comment:
Let's use `tx.Enum` so that the validation happens earlier, by defining an enum class to represent the available operation types.
(ref: https://github.com/lablup/backend.ai-common/blob/main/src/ai/backend/common/validators.py#L171)

Reply:
added t.Enum('and', 'or') for thresholds-check-operator

Reply:
Changed Operator class to ThresholdOperator